src/app/buy-credits/buy-credits.component.html

```diff
@@ -0,0 +1,150 @@
+<main>
+    <div *ngIf="isLoggedIn" id="loggedInHeader">
+        <p>Logged in as {{ userFullName }}</p>
+    </div>
+    <div class="b-container">
+        <h3 class="b-rh-1 b-bold">Purchase Donation Credits</h3>
+
+        <form *ngIf="isLoggedIn && !isLoading && !isPurchaseComplete" [formGroup]="creditForm">
+            <mat-stepper orientation="vertical" [linear]="isLinear" #stepper>
+                <mat-step
+                    [completed]="false"
+                    formGroupName="amounts"
+                    [stepControl]="amountsGroup"
+                >
+                    <ng-template matStepLabel>Your Donation Credits</ng-template>
+                    <mat-form-field  class="form-field b-center size-lg" hideRequiredMarker>
+                        <mat-label>Credit amount (£)</mat-label>
```

Comment:
Any reason not to make this a public class var? We pretty much eliminated currency hard-coding in templates across the whole app, at some length, so it seems a shame to bring it back.